src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -501,41 +412,27 @@ function EmojiPickerMenu(props) {
                     blurOnSubmit={filteredEmojis.length > 0}
                 />
             </View>
-            {!isFiltered && (
-                <CategoryShortcutBar
-                    headerEmojis={headerEmojis}
-                    onPress={scrollToHeader}
-                />
-            )}
-            <FlatList
-                ref={emojiListRef}
-                data={filteredEmojis}
-                renderItem={renderItem}
-                keyExtractor={keyExtractor}
-                numColumns={CONST.EMOJI_NUM_PER_ROW}
-                style={[
+            <BaseEmojiPickerMenu
+                isFiltered={isListFiltered}
+                headerEmojis={headerEmojis}
+                scrollToHeader={scrollToHeader}
+                listWrapperStyle={[
                     listStyle,
-                    // This prevents elastic scrolling when scroll reaches the start or end
-                    {overscrollBehaviorY: 'contain'},
```

Comment:
Can we be sure to retest the [original bug](https://github.com/Expensify/App/issues/22939#issuecomment-1655555722) in iOS Safari? Apparently that's where it occurs: https://expensify.slack.com/archives/C01GTK53T8Q/p1702629684965709?thread_ts=1702597102.525129&cid=C01GTK53T8Q

Reply:
I couldn't reproduce it - seems like it's safe to remove it